### PR TITLE
docs(v5): kanad edits

### PR DESCRIPTION
**Pull request by kanadgupta**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -4,39 +4,41 @@ category: 5d4c940cae4e610060475769
 ---
 
 <p align="center">
-  <img width="350" src="./images/logo.svg" />
```
Comment (kanadgupta) [inline docs/getting-started.md:7]: Most of the changes in this file are just to get it up to par with the main `README.md` file, but I think we should do some reuse if possible. See #486

File: docs/getting-started.md [hunk 2]
```diff
@@ -4,39 +4,43 @@ category: 5d4c940cae4e610060475769
...
+  <img width="400" src="./images/logo.svg" />
 </p>
 
 <p align="center">
-  Automatic SDK generation from an OpenAPI definition.
+  Magical SDK generation from an OpenAPI definition 🪄
```
Comment (erunion) [inline docs/getting-started.md:11]: wanna update the description in the package file too?

File: docs/getting-started.md [hunk 3]
```diff
@@ -4,39 +4,43 @@ category: 5d4c940cae4e610060475769
...
+$ npx api install https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore.json
 ```
 
 ```js
-const sdk = require('@api/petstore');
+const SDK = require('@api/petstore');
 
-sdk.listPets().then(res => {
+const petstore = new SDK();
+petstore.listPets().then(res => {
```
Comment (erunion) [inline docs/getting-started.md:31]: prettier will break this up into three lines i think

File: docs/how-it-works.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ The `.api/` directory is wher the CLI installation process stores all of its inf
     ├── readme/
     |   ├── node_modules/
     │   ├── index.js
-    │   ├── index.d.ts   // All types for the SDK, ready to use in your an IDE.
+    │   ├── index.d.ts   // All types for the SDK, ready to use in your IDE.
```
Comment (erunion) [inline docs/how-it-works.md:39]: I might have this same typo in the storage library

File: docs/parameters-and-payloads.md [hunk 5]
```diff
@@ -34,11 +34,11 @@ Since we've supplied two objects here, the SDK automatically knows that you're s
 
 > ℹ️
 >
-> For background, the algorithm to determine if you're supplying a `body` or `metadata` for the first parameter runs a difference check against the payloads for either and if what you're supplying matches +25% of what is documented in the operations request body payload then we'll treat it as a `body` payload.
+> For background, the algorithm to determine if you're supplying a `body` or `metadata` for the first parameter runs a difference check against the payloads for either. If what you're supplying matches +25% of what is documented in the operation's request body payload, then we'll treat it as a `body` payload.
```
Comment (erunion) [inline docs/parameters-and-payloads.md:37]: Should "operation's" have an apostrophe in this context?

File: docs/server-configurations.md [hunk 6]
```diff
@@ -3,18 +3,18 @@ title: Server Configurations
 category: 62cc6ce22b8b6601da6cb12d
 ---
 
-If the API you're using offers alternate server URLs and server variables in its [`servers`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#serverObject) definition you can supply this to the SDK with `.server()`:
+If the API you're using offers alternate server URLs and server variables in its [`servers`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#serverObject) definition, you can supply this to the SDK with `.server()`:
 
 ```js
 sdk.server('https://{region}.api.example.com/{basePath}', {
-  name: 'eu',
+  region: 'eu',
```
Comment (erunion) [inline docs/server-configurations.md:10]: 👀 oops

**Review by erunion — APPROVED** [review]
surprised at how few fixes you had to make! :aw-thanks:

File: docs/getting-started.md (hunk 3 above)
Comment (kanadgupta) [reply docs/getting-started.md:31]: Prettier appears to approve: https://github.com/readmeio/api/runs/7332391830?check_suite_focus=true#step:7:7

File: docs/how-it-works.md (hunk 4 above)
Comment (kanadgupta) [reply docs/how-it-works.md:39]: Did a quick search and couldn't find anything!

File: docs/parameters-and-payloads.md (hunk 5 above)
Comment (kanadgupta) [reply docs/parameters-and-payloads.md:37]: I think so? The way I'm reading this, the request body payload is possessed by the operation

File: docs/getting-started.md (hunk 2 above)
Comment (kanadgupta) [reply docs/getting-started.md:11]: done: https://github.com/readmeio/api/pull/487/commits/52bdd7791374ab400fc1fbf5a2ac59104f868fcf

**Pull request merged**